packages/node/test/integration/propose-install.spec.ts

```diff
@@ -29,7 +29,7 @@ describe("Node method follows spec - proposeInstall", () => {
   let nodeB: Node;
   let storeServiceB: IStoreService;
   let nodeConfig: NodeConfig;
-  let provider: BaseProvider;
+  let provider: JsonRpcProvider;
```

Comment:
can you split out all the `BaseProvider` -> `JsonRpcProvider` changes

Reply:
sure